packages/skeleton-react/src/lib/components/Navigation/Navigation.test.tsx

```diff
@@ -128,7 +128,7 @@ describe('Navigation', () => {
 		it('Renders the component', () => {
 			const { getByTestId } = render(
 				<Navigation.Rail>
-					<Navigation.Tile></Navigation.Tile>
+					<Navigation.Tile id="0"></Navigation.Tile>
```

Comment:
@phamduylong the change to use `useId()` is great Long, but we're missing the benefit to this change if we still show all our examples including an `id`. This should now be an optional field, so let's review these from the examples. The API reference will indicate this is available if folks wish to add it.

To put it another way, we try to keep examples a minimal as possible so folks can focus more on what is there, than what isn't.